qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -301,8 +298,8 @@ class AdmissionControllerTest extends BaseSpecification {
         "Data inputs are: "
 
         image | _
-        NGINX_IMAGE_WITH_SHA | _
-        NGINX_IMAGE | _
+        TEST_IMAGE | _
+        TEST_IMAGE.split("@").first() | _
```

Comment:
Instead of relying on `TEST_IMAGE` having a particular format, perhaps it would make sense to have a `TEST_IMAGE_WITH_SHA` and `TEST_IMAGE_WITHOUT_SHA` constants instead? 🤔

Reply:
Reverted changes to this file